sdk/metric/sdk.go

```diff
@@ -462,7 +462,7 @@ func (m *Accumulator) checkpointAsync(a *asyncInstrument) int {
 		epochDiff := m.currentEpoch - lrec.observedEpoch
 		if epochDiff == 0 {
 			if lrec.observed != nil {
-				exportRecord := export.NewRecord(&a.descriptor, lrec.labels, m.resource, lrec.observed)
+				exportRecord := export.NewAccumulation(&a.descriptor, lrec.labels, m.resource, lrec.observed)
 				err := m.integrator.Process(exportRecord)
```

Comment:
Update var name to avoid type confusion.

```suggestion
				a := export.NewAccumulation(&a.descriptor, lrec.labels, m.resource, lrec.observed)
				err := m.integrator.Process(a)
```